scripts/collector/mysql/collect-data.sh

```diff
@@ -274,19 +274,37 @@ then
 	DMA_SOURCE_ID="NA"
 fi
 
-PGPASSWORD="$pass"  ${SQLCMD}  --user=$user -d $db -h $host -w -p $port  --no-align <<EOF
-\set VTAG '\'${V_FILE_TAG}\''
+echo ${SQLCMD} -X --user=$user -d $db -h $host -w -p $port  --no-align
+
+PGPASSWORD="$pass"  
```

Comment:
No need to prefix export here, correct? It should be set for the duration of the script?